libs/shared/contentful/README.md

```diff
@@ -0,0 +1,11 @@
+# shared-contentful
+
+This library was generated with [Nx](https://nx.dev).
```

Comment:
Would be good to mention that this is a basic library for talking to Contentfuls graphql API.